source/org/zfin/infrastructure/presentation/PdbLinkController.java

```diff
@@ -0,0 +1,42 @@
+package org.zfin.infrastructure.presentation;
+
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.zfin.framework.presentation.LookupStrings;
+import org.zfin.sequence.ProteinToPDB;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.zfin.repository.RepositoryFactory.getMarkerRepository;
+
+@Controller
+@RequestMapping("/infrastructure")
+@Log4j2
+public class PdbLinkController {
+
+    public static final String PDB_URL = "https://rcsb.org/structure/";

```

Comment:
Could you get this URL from the foreign_db record of the database?

Reply:
Good idea. I updated it. Thanks